java/src/jmri/jmrix/loconet/messageinterp/MessageInterpretation.properties

```diff
@@ -1554,6 +1554,7 @@ LN_MSG_ON  = On
 LN_MSG_OFF = Off
 LN_MSG_COMMAND_STATION = (CS)
 
+LN_MSG_EXTEND_ACCY_SET_ASPECT = Extended Accessory Decoder Set Digitrax [{0}] NMRA [{1}] to Aspect [{2}].\n
```

Comment:
Need "Address"es.  Never seen address value encapsulated in square brackets.  How about:

LN_MSG_EXTEND_ACCY_SET_ASPECT = Extended Accessory Decoder Set Digitrax Addsres {0} (NMRA Address {1}) to Aspect {2}.\n